pages/common/git-credential.md

```diff
@@ -0,0 +1,16 @@
+# git credential
+
+> Retrieve and store user credentials.
+> More information: <https://git-scm.com/docs/git-credential>.
+
+- Display credential information, retrieving the username/password from config files:
+
+`echo {{"url=http://example.com"}} | git credential fill`
```

Comment:
I think the quotes should be outside the token syntax here - same goes for the other examples.